lib/IRGen/IRGenDebugInfo.cpp

```diff
@@ -3869,6 +3869,14 @@ void IRGenDebugInfoImpl::emitGlobalVariableDeclaration(
   if (Opts.DebugInfoLevel <= IRGenDebugInfoLevel::LineTables)
     return;
 
+  // If this global variable contains an opaque type, replace it with its
+  // underlying type.
+  if (auto UnderlyingType = IGM.substOpaqueTypesWithUnderlyingTypes(
```

Comment:
substOpaqueTypesWithUnderlyingTypes never returns null, so this if statement is always taken

Reply:
And it's a noop for all other types?